vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/dataview/GridListDataView.java

```diff
@@ -57,4 +58,9 @@ public int getSize() {
         return dataCommunicator.getDataSize();
     }
 
+    @Override
+    public void setIdentifierProvider(IdentifierProvider<T> identifierProvider) {
```

Comment:
Now this method is not used/tested from anywhere, making it possible to remove it without any tests to fail

Reply:
It is tested in `GridDataViewImplTest`, however the code is masked a bit: 
```
        // In-memory grid data view
        dataView = component.getListDataView();
```
But it should be in the `GridListDataViewTest` apparently.

Reply:
Well if it is tested and cannot be removed accidentally then it is enough